dhcpv6/option_userclass_test.go

```diff
@@ -91,3 +91,13 @@ func TestOptUserClassString(t *testing.T) {
 		"String() should contain the list of user classes",
 	)
 }
+
+func TestOptUserClassParseOptUserClassCiena(t *testing.T) {
```

Comment:
What is this additional test for? Is it covering something we were not before?

Reply:
Stale experiment, let me remove it :-)